src/screens/CreateNFT.js

```diff
@@ -262,8 +263,8 @@ function CreateNFT() {
   // The actual math for the UI value would be htrDeposit * 100 / (10^(2-DECIMAL_PLACES))
   // Since NFT have 0 for DECIMAL_PLACES we get htrDeposit * 100/100 which is htrDeposit
   const htrDeposit = wallet.storage.getTokenDepositPercentage();
-  const depositAmount = hathorLib.tokensUtils.getDepositAmount(amount, htrDeposit);
-  const nftFee = hathorLib.numberUtils.prettyValue(tokensUtils.getNFTFee());
+  const depositAmount = hathorLib.tokensUtils.getDepositAmount(amount, htrDeposit, decimalPlaces);
```

Comment:
This method from the lib (https://github.com/HathorNetwork/hathor-wallet-lib/blob/master/src/utils/tokens.ts#L269) does not have this new parameter. We don't need it, actually, because it returns a number.

Reply:
Changed